Sources/message/ALMessage.m

```diff
@@ -296,20 +296,16 @@ - (NSString *)getVOIPMessageText {
 }
 
 - (BOOL)isMsgHidden {
-    BOOL hide = [[self.metadata objectForKey:@"hide"] boolValue];
-
-    // Check messages that we need to hide
-    NSArray *keys = [ALApplozicSettings metadataKeysToHideMessages];
```

Comment:
Is this not required?

Reply:
its not required.because this logic hides the message if it has assignment key in meta data(its causing the handoff message to be hidden). But in web they improved the logic so that hand-off message won't get affected.